geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9990,6 +10011,43 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  PartitionRegionConfig updatePRNodeInformation() {
```

Comment:
The return value of type `PartitionRegionConfig` is never used in the product. It is only used in the test though.  How about using `void`?

Reply:
Test requires access to the new configuration for verification purposes.
No way to determine new configuration without this return value.
The return value does not impact product code, so leaving as is.

Reply:
With the new code change, we update the `Node` on the spot with the existing `prConfig`, without creating a `newConfig`. The return value is not necessary. To get the latest `PartitionRegionConfig`, `PartitionRegionConfig prConfig = getPRRoot().get(getRegionIdentifier());` should work. Therefore, the return type of `updatePRNodeInformation` should be `void`.